corehq/tabs/tabclasses.py

```diff
@@ -1153,7 +1153,7 @@ def _is_viewable(self):
         return (
             has_privilege(self._request, privileges.CLOUDCARE)
             and self.domain
-            and (self.couch_user.can_access_any_web_apps() or self.couch_user.is_commcare_user())
+            and self.couch_user.can_access_any_web_apps()
```

Comment:
this feels really satisfying

Reply:
reverting this change, see longer comment below